inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -884,8 +884,15 @@ std::map<const ngraph::DiscreteTypeInfo, std::function<void(const std::shared_pt
     {ngraph::op::v0::Clamp::type_info, [](const std::shared_ptr<ngraph::Node>& op, MKLDNNEltwiseNode& node) {
         auto clampOp = getNgraphOpAs<ngraph::op::v0::Clamp>(op);
 
-        node.alpha = static_cast<float>(clampOp->get_min());
-        node.beta = static_cast<float>(clampOp->get_max());
+        float alpha_ = static_cast<float>(clampOp->get_min());
+        float beta_ = static_cast<float>(clampOp->get_max());
+        if (one_of(clampOp->get_input_element_type(0), ngraph::element::i8, ngraph::element::i32, ngraph::element::i64,
+                   ngraph::element::u64)) {
```

Comment:
Why is there only one unsigned int - i64?

Maybe it will be better this way?
```suggestion
        if (clampOp->get_input_element_type(0)is_integral_number()) {
```